src/main/java/uk/gov/pay/ledger/app/LedgerModule.java

```diff
@@ -156,4 +157,4 @@ public SqsQueueService provideSqsQueueService(AmazonSQS amazonSQS, LedgerConfig
                 ledgerConfig.getSqsConfig().getMessageMaximumWaitTimeInSeconds(),
                 ledgerConfig.getSqsConfig().getMessageMaximumBatchSize());
     }
-}
+}
```

Comment:
Dude, your IDE is at war with the Unix convention that text files end with a newline.

Reply:
Thanks for flagging, found and unset the rogue setting, no ideological beef here